kolibri/plugins/coach/assets/src/views/common/commonCoachStrings.js

```diff
@@ -217,7 +212,6 @@ const coachStrings = createTranslator('CommonCoachStrings', {
     context:
       "Coaches can choose between 'Randomized' and 'Fixed' question order when they create quizzes.\n\nThis text is a description of the 'Randomized' question order.",
   },
-  overallLabel: 'Overall',
```

Comment:
String used in old designs, [not useful anymore](https://learningequality.slack.com/archives/C0LKG14NL/p1631145468008100).